deps/cares/include/ares_build.h

```diff
@@ -0,0 +1,117 @@
+/* ares_build.h.  Generated from ares_build.h.in by configure.  */
```

Comment:
Should this file be defined in the GYP file too? Also, ares_rules.h was moved: https://github.com/nodejs/node/blob/master/deps/cares/cares.gyp#L82